resources/styles/components/exercises.less

```diff
@@ -104,21 +138,27 @@
       line-height: @size;
 
       &.section {
-        border: 1px solid @tutor-neutral-light;
-        min-height: @size;
-        &+.section {
-          border-left: 0;
+        border: 1px solid @default-border;
+        height: @size;
+        position: relative;
+        margin-right: 0;
+        margin-left: -1px;
+
+        &:first-child {
+          margin-left: 0;
         }
       }
       &.active {
         background-color: @tutor-neutral-light;
+        z-index: 10;
       }
```

Comment:
Hm, what's the `z-index` needed for?  I **seems** like an active sections shouldn't be floated above anything